src/RestApi/V1/Resource/MerchantIntegrationsResource.php

```diff
@@ -20,9 +20,7 @@ class MerchantIntegrationsResource implements MerchantIntegrationsResourceInterf
     /**
      * @internal
      */
-    public function __construct(private readonly PayPalClientFactoryInterface $payPalClientFactory)
-    {
-    }
+    public function __construct(private readonly PayPalClientFactoryInterface $payPalClientFactory) {}
```

Comment:
all those parentheses, keep them in separate lines. I don't want to differ here from platform!

Reply:
your wish is my command